### PR TITLE
fix(gulp): add babel runtime options

### PR DESCRIPTION
**Pull request by david-mohr**
without `es7.classProperties`, babel won't compile the default controller template

### CODE REVIEW

**Comment by Awk34** [comment]
I use the `'gulp-load-plugins'` module, which finds all npm modules we use that start with `gulp-` and adds them on the `plugins` object that I have at the top of the file; so you don't have to change `plugins.babel` to `babel`.

File: app/templates/gulpfile.babel(gulp).js [hunk 1]
```diff
@@ -392,7 +395,7 @@ gulp.task('build:client', ['transpile:client', 'styles', 'html'], () => {
 
     let assets = plugins.useref.assets({searchPath: ['client', '.tmp']});
 
-    return gulp.src(paths.mainView)<% if(filters.jade) { %>
```
Comment (Awk34) [inline app/templates/gulpfile.babel(gulp).js]: split this change out to a separate commit, since it's unrelated to the other changes

Comment (david-mohr) [inline app/templates/gulpfile.babel(gulp).js]: Would you like a new PR, or just a separate commit? (sorry, pretty new to this)

Comment (Awk34) [inline app/templates/gulpfile.babel(gulp).js]: Just pull it into a separate commit. If you know how to rebase, I'd suggest it. If not, just make a new PR.

**Comment by Awk34** [comment]
Thanks!
